hbase-client/src/main/java/org/apache/hadoop/hbase/client/TableDescriptorBuilder.java

```diff
@@ -380,6 +380,10 @@ public TableDescriptorBuilder setCoprocessors(Collection<CoprocessorDescriptor>
     return this;
   }
 
+  public boolean hasCoprocessor(String classNameToMatch) {
```

Comment:
Adding method to a public interface. For hbase3 only so should be fine. This is filling in missing functionality?

Reply:
This is a class, not an interface so I think it is OK? I think for a builder class it is also OK to have some 'get' methods to know what have we set into the builder?